en/sql/transaction.rst

```diff
@@ -802,6 +802,14 @@ CUBRID determines the lock mode depending on the type of operation to be perform
     | requesting SCH_M_LOCK but waiting T2's SCH_S_LOCK is released |                                                               |
     +---------------------------------------------------------------+---------------------------------------------------------------+
 
+*   **Special Locks**
+
+    **CUBRID 10.2** introduces a new type of locks, which are used in a few special cases of internal operations.
+
+    *   **Bulk Update Lock, BU_LOCK**
+
+        This lock is designed for inserting large amounts of data into the database. As of **CUBRID 10.2**, this lock is exclusively used during the :ref:`loaddb` process while loading rows into a table. **BU_LOCK** is compatible with itself and **SCH_S_LOCK**; as a result this ensures that no other **SELECT/UPDATE/DDL** statements are allowed on the same table. However, multiple **loaddb** may load rows into a table. The **BU_LOCK** holder doesn't require row locks.
```

Comment:
**SELECT/UPDATE/DDL** doesn't include **DELETE/INSERT** statements :). I think **SELECT/DML/DDL** would be more accurate.